plugins/ExtendedTools/objmgr.c

```diff
@@ -2264,7 +2264,7 @@ VOID NTAPI EtpObjectManagerObjectProperties(
     }
 
     objectContext.CurrentPath = PhReferenceObject(context->CurrentPath);
-    objectContext.Object = PhReferenceObject(Entry);
+    objectContext.Object = Entry;
```

Comment:
This can be rewritten with:

```cpp
 PhSetReference(&objectContext.CurrentPath, context->CurrentPath);
 PhSetReference(&objectContext.Object, Entry);
```

PhSetReference checks for null or references the object.

Reply:
In current implementation we cannot reference for `Entry` here, because it can be fake Directory object on stack. `ET_CREATE_DIRECTORY_OBJECT_ENTRY` macro fill necessary fields of `ET_OBJECT_ENTRY` to work with directory through common functions. Currently, the tree only holds directory string in item lParam, but not and object entry.
I added this code by mistake.